lib/ocpp/v201/charge_point.cpp

```diff
@@ -673,7 +692,6 @@ void ChargePoint::init_websocket() {
             WEBSOCKET_INIT_DELAY);
         return;
     }
-
     const auto& security_profile_cv = ControllerComponentVariables::SecurityProfile;
```

Comment:
You are not checking if it was successful before running this code.

And I think it's good to add a TODO here (maybe including a ticket number) that we have to reconnect the websocket with the given interface address / name.